tests/functional_tests/ffront_tests/test_program.py

```diff
@@ -278,3 +278,48 @@ def empty_domain_program(
         match=(r"Dimensions in out field and field domain are not equivalent"),
     ):
         empty_domain_program(a, out_field, offset_provider={})
+
+
+def test_input_kwargs(fieldview_backend):
+    size = 10
+    input_1 = np_as_located_field(IDim, JDim)(np.ones((size, size)))
+    input_2 = np_as_located_field(IDim, JDim)(np.ones((size, size)) * 2)
+    input_3 = np_as_located_field(IDim, JDim)(np.ones((size, size)) * 3)
+    out = np_as_located_field(IDim, JDim)(np.zeros((size, size)))
+
+    expected_fieldop = np.asarray(input_3) * np.asarray(input_1) - np.asarray(input_2)
+    expected_program = np.asarray(input_3) * np.asarray(input_2) - np.asarray(input_1)
+
+    @field_operator(backend=fieldview_backend)
+    def fieldop_input_kwargs(
+        a: Field[[IDim, JDim], float64],
+        b: Field[[IDim, JDim], float64],
+        c: Field[[IDim, JDim], float64],
+    ) -> Field[[IDim, JDim], float64]:
+        return c * a - b
+
+    fieldop_input_kwargs(input_1, b=input_2, c=input_3, out=out, offset_provider={})
+    assert np.allclose(expected_fieldop, out)
+
+    @program
+    def program_input_kwargs(
+        a: Field[[IDim, JDim], float64],
+        b: Field[[IDim, JDim], float64],
+        c: Field[[IDim, JDim], float64],
+        out: Field[[IDim, JDim], float64],
+    ):
+        fieldop_input_kwargs(c, a, b, out=out)
+
+    out = np_as_located_field(IDim, JDim)(np.zeros((size, size)))
+
+    program_input_kwargs(input_1, b=input_2, c=input_3, out=out, offset_provider={})
+    assert np.allclose(expected_program, out)
+    out = np_as_located_field(IDim, JDim)(np.zeros((size, size)))
```

Comment:
```suggestion
    out = np_as_located_field(IDim, JDim)(np.zeros((size, size)))
    program_input_kwargs(input_1, b=input_2, c=input_3, out=out, offset_provider={})
    assert np.allclose(expected_program, out)
```
If you reinitialize the out field please use this grouping everywhere.